src/app/backend/handler/apihandler.go

```diff
@@ -61,6 +57,10 @@ import (
 	errorsK8s "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/client/unversioned/remotecommand"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
```

Comment:
@floreks goimports changes about 20 files. Should I change as part of this PR? WDYT?